app/services/orchestration_template_cfn_dialog_service.rb

```diff
@@ -0,0 +1,42 @@
+class OrchestrationTemplateCfnDialogService < OrchestrationTemplateDialogService
+  private
+
+  def add_deployment_options(dialog_group, position)
+    add_on_failure_field(dialog_group, position)
+    add_timeout_field(dialog_group, position + 1)
+  end
+
+  def add_on_failure_field(group, position)
+    group.dialog_fields.build(
+      :type          => "DialogFieldDropDownList",
+      :name          => "stack_onfailure",
+      :description   => "Select what to do if stack creation failed",
+      :data_type     => "string",
+      :display       => "edit",
+      :required      => true,
+
+      # %w(DELETE Delete\ stack) is available with aws-sdk v2
```

Comment:
I noticed this was in the code before, but just didn't understand why the comment is here? Should this be a TODO once v2 of aws-sdk is used? If so, I'd add TODO or even check whether the version currently used (Gemfile says `gem "aws-sdk", "~>2"`) could now use this option as well.